src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -410,6 +530,15 @@ let create (aggSpec: AggregatorSpec, aggArgs: AggregatorArgs) : T =
     else
       0
 
+  let unpackConstInt index =
```

Comment:
I got stuck on this for a while, somehow thinking there's much more to it than it actually is. Given the error message is specific anyway, maybe better to call this function specifically like `unpackGeneralizationWidthArgAt`.